cmd/orb-server/startcmd/start.go

```diff
@@ -354,9 +354,11 @@ func startOrbServices(parameters *orbParameters) error {
 	}
 
 	useHTTPOpt := false
+	webFingerURIScheme := "https"
 
 	if parameters.tlsKey == "" && parameters.tlsCertificate == "" {
 		useHTTPOpt = true
+		webFingerURIScheme = "http"
```

Comment:
You shouldn't be coupling having TLS configured on the orb server to having non-https on the client.

You can not have TLS certificates by terminating HTTPS on a different system and still be in HTTPS mode.

Use a different argument to signal the HTTPS on the client.

Reply:
done. added new flag